packages/wrangler/e2e/unenv-preset/worker/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
have you considered potentially disabling the rule only from line 371 to line 376 instead? is it worth it disabling the rule for the whole file when it's only needed in that block of code?

Reply:
Since it is in a test file it's not a huge deal to me either 🙂👍